integration/airflow/openlineage/airflow/extractors/base.py

```diff
@@ -56,8 +56,8 @@ def validate(self):
         assert (self.operator.__class__.__name__ in self.get_operator_classnames())
 
     @abstractmethod
-    def extract(self) -> Optional[TaskMetadata]:
+    def extract(self, task_instance=None) -> Optional[TaskMetadata]:
```

Comment:
I think the main reason behind `TaskInstance` not being here is previous way of integrating with Airflow using subclassed DAG. We can enable it now, but if we do, I think we should update OpenLineageBackend and OpenLineagePlugin to pass it.